dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolReferenceSet.java

```diff
@@ -20,14 +20,14 @@
 import com.google.common.collect.ImmutableSet;
 
 /**
- * Symbolic references in a file (jar file or class file). The references are from constant pool of
- * class file(s).
+ * A set of symbolic references in a file (jar file or class file). The references are from constant
```

Comment:
I'd say "symbol references" instead of "symbolic references". They're references to symbols, not references that are symbolic.

Reply:
Updated them.